README.md

```diff
@@ -1,33 +1,33 @@
 # AnimatedTextInput
-iOS custom text input component used in Jobandtalent app.
+iOS custom text input component used in the [Jobandtalent app](https://itunes.apple.com/app/id665060895).
 
-![](https://github.com/jobandtalent/AnimatedTextInput/blob/master/Src/general.gif)
+![](https://github.com/jobandtalent/AnimatedTextInput/blob/master/Assets/general.gif)
 
 ## Installation
 Use cocoapods to install this custom control in your project.
 
 ```
-pod 'AnimatedTextInput'
+pod 'AnimatedTextInput', '~> 0.1.0'
```

Comment:
Chances are this will get out of sync pretty soon if we don't remember to update the README when we update the podspec. Do we really need to specify the version here?